deepchem/models/torch_models/prot_bert.py

```diff
@@ -120,7 +128,30 @@ def __init__(self,
             model = BertForSequenceClassification.from_pretrained(
                 model_path, config=protbert_config)
             model.classifier = cls_head
+        elif task == "feature_extractor":
+            model = BertModel.from_pretrained(model_path,
+                                              config=protbert_config)
 
         else:
             raise ValueError('Invalid task specification')
         super().__init__(model=model, task=task, tokenizer=tokenizer, **kwargs)
+
+    def get_feat(self, input_ids: torch.Tensor,
```

Comment:
I think our convention has been to use predict_embedding but please take a second look

Reply:
Renamed to predict_embedding